bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeEphemerisFeature.scala

```diff
@@ -188,7 +188,7 @@ object TpeEphemerisFeature {
   }
 
   val formatDate: Long => String = {
-    val df = new SimpleDateFormat("dd-MMM HH:mm")
+    val df = new SimpleDateFormat("yyyy-MMM-dd HH:mm")
```

Comment:
This one writes on the TPE display on top of everything else so it might be better to keep it short, even if technically inconsistent?  That is maybe just switch the `dd-MMM` as you've done but leave off the year?